src/main/resources/entity-types/inventory/composite_instances.json5

```diff
@@ -16,7 +16,7 @@
       join: {
         type: 'left join',
         joinTo: 'instance',
-        condition: ":this.id = (:that.jsonb ->> 'statusId')::uuid"
+        condition: ":this.id = :that.instanceStatusId"
```

Comment:
These join condition changes are just something I noticed when adding the new source. Apparently the `instance` table has indexes on all of its non-jsonb columns and not on the corresponding jsonb fields, so we should see a pretty big performance boost from this change